openmmtools/forces.py

```diff
@@ -267,35 +283,65 @@ class RadiallySymmetricRestraintForce(utils.RestorableOpenMMObject):
 
     def __init__(self, restraint_parameters, restrained_atom_indices1,
                  restrained_atom_indices2, controlling_parameter_name,
+                 energy_string,
                  *args, **kwargs):
+        # Ensure parameter names are in the correct format
+        assert len(restraint_parameters) == 1 or isinstance(restraint_parameters, collections.OrderedDict)
+        assert "Distance" not in restraint_parameters, ('"Distance" cannot be a restraint_parameter as it will ' 
+                                                        'clash with the CV Variable"')
+
+        # Augment energy_string with parameters
+        energy_string += ";"
+        for parameter_name, parameter_value in restraint_parameters.items():
+            # Ensure OpenMM Unit system
+            if isinstance(parameter_value, unit.Quantity):
+                parameter_value = parameter_value.in_unit_system(unit.md_unit_system)
+                restraint_parameters[parameter_name] = parameter_value
+                # Strip unit without overwriting dict entry efficiently
+                # Fetching value is faster than dividing by unit since
+                # we are already in correct unit system
+                parameter_value = parameter_value._value
+            energy_string += "{} = {};".format(parameter_name, parameter_value)
+        # Augment args with the energy string
+        args = (energy_string,) + args
+
+        # Construct
         super(RadiallySymmetricRestraintForce, self).__init__(*args, **kwargs)
         self._controlling_parameter_name = controlling_parameter_name
 
-        # Unzip bond parameters names and values from dict.
-        assert len(restraint_parameters) == 1 or isinstance(restraint_parameters, collections.OrderedDict)
-        parameter_names, parameter_values = zip(*restraint_parameters.items())
+        # Create the distance force used for the CV force wraps
+        distance_force = self._create_distance_force()
+        self.addCollectiveVariable("Distance", distance_force)
```

Comment:
What should our naming convention be for these CVs? 
* Do we want to capitalize them (`Distance`) or make them lowercase (`distance`)? 
* Be more explicit in the names (e.g. `receptor_ligand_distance`) or use math symbols (`r`, `theta1`, etc.)?

Reply:
I prefer the capital "Distance` because it clearly indicates this is a unique variable the user must think about. I don't want to use `r` or `distance(g1,g2)` here because this parent class is agnostic of the underlying CV implementation.

Reply:
Have we thought through what names we will use for CVs for all the other restraints?

Reply:
Right now the only other restraint classes I have are the Boresch and RMSD ones. 

The RMSD one's have the CV `RMSD`
Boresch has CVs `Distance`, `AngleA`, `AngleB`, `TorsionA`/`B`/`C.

There is overlap there, yes, but the way I implemented the `SamplerState` CV checking is supports multiple forces having overlapping CV Names. I see the overlap here. I'm open to new name ideas, its very easy to change at this point since I have not implemented any of the analysis to take advantage of this conversion yet.

Reply:
[Here](https://github.com/choderalab/yank/blob/96249bcd7b26a779df880052de3813afa1bad3c1/Yank/restraints.py#L1639-L1682) is the reference implementation of the Boresch restraints are in YANK at the moment.